src/main/java/com/terraformersmc/cinderscapes/block/PolypiteQuartzBlock.java

```diff
@@ -65,9 +65,9 @@ public BlockState stateOf(Direction dir) {
     static {
         DIRECTION_TO_SHAPE.put(Direction.DOWN, Block.createCuboidShape(2.0D, 0.0D, 2.0D, 14.0D, 4.0D, 14.0D));
         DIRECTION_TO_SHAPE.put(Direction.UP, Block.createCuboidShape(2.0D, 12.0D, 2.0D, 14.0D, 16.0D, 14.0D));
-        DIRECTION_TO_SHAPE.put(Direction.SOUTH, Block.createCuboidShape(0.0D, 4.0D, 5.0D, 16.0D, 12.0D, 16.0D));
-        DIRECTION_TO_SHAPE.put(Direction.NORTH, Block.createCuboidShape(0.0D, 4.0D, 0.0D, 16.0D, 12.0D, 11.0D));
-        DIRECTION_TO_SHAPE.put(Direction.EAST, Block.createCuboidShape(5.0D, 4.0D, 0.0D, 16.0D, 12.0D, 16.0D));
-        DIRECTION_TO_SHAPE.put(Direction.WEST, Block.createCuboidShape(0.0D, 4.0D, 0.0D, 11.0D, 12.0D, 16.0D));
+        DIRECTION_TO_SHAPE.put(Direction.SOUTH, Block.createCuboidShape(3.0D, 5.0D, 6.0D, 13.0D, 13.0D, 16.0D));
```

Comment:
Could you supply a visual before and after of what this change does to the bounding box of the quartz.

Reply:
Before:
![before](https://user-images.githubusercontent.com/36963654/86544609-a4a38b80-bf28-11ea-8efd-e1cb38909f6b.png)
After:
![after](https://user-images.githubusercontent.com/36963654/86544608-a3725e80-bf28-11ea-84eb-5b15f443d7ce.png)